lib/Gocdb_Services/ServiceService.php

```diff
@@ -2083,6 +2081,23 @@ public function EndpointPropSet (\EndpointLocation $endpoint, $endpointPropName)
 
     }
 
+    /**
+     * Returns true if an endpoint with a given name exists for a given service
+     *
+     * TODO: This could be made more effecient using a DQL select statement
```

Comment:
```suggestion
     * TODO: This could be made more efficient using a SQL select statement
```

Reply:
Clarify what DQL is?

Reply:
Doctrine Query Language - used elsewhere in codebase